main.ts

```diff
@@ -121,34 +130,120 @@ class TodoistPluginSettingTab extends PluginSettingTab {
 							await this.plugin.saveSettings();
 						}
 					));
+		containerEl.createEl('hr').addClass("dashed");
 	}
 
 	private addIncludeSubttasksSetting(containerEl: HTMLElement) {
+		/* 
+			Subtask Handling
+		*/
+		containerEl.createEl('h2', {text: 'Subtask Handling'}).addClass("heading");
 		new Setting(containerEl)
 			.setName('Enable Subtasks')
-			.setDesc("When enabled, any Subtasks associated with Todos meeting filter criteria from your keyword will be shown, indented, under the parent Todo.")
+			.setDesc("When enabled, any Subtasks associated with Tasks meeting filter criteria from your keyword will be shown, indented, under the parent Task.")
 			.addToggle(t =>
 				t.setValue(this.plugin.settings.showSubtasks)
 					.onChange(async (value) => {
 							this.plugin.settings.showSubtasks = value;
+							if (this.plugin.settings.showSubtasks == false) {
+								subtaskDiv.toggleClass('hide',true);
+							} else {
+								subtaskDiv.toggleClass('hide',false);
+							}
+							await this.plugin.saveSettings();
+						}
+					));
+		const subtaskDiv = containerEl.createEl("div");
+		subtaskDiv.addClass("todoist-setting-div");
+		new Setting(subtaskDiv)
+			.setName('Include Subtasks with no due dates')
+			.setDesc("This includes showing subtasks with no due dates that are associated with any parent tasks included in the filter criteria. On by default.")
+			.setClass("todoist-setting-top")
+			.addToggle(t =>
+				t.setValue(this.plugin.settings.noDateSubtasks)
+					.onChange(async (value) => {
+							this.plugin.settings.noDateSubtasks = value;
+							await this.plugin.saveSettings();
+						}
+					));
+		new Setting(subtaskDiv)
+			.setName('Limit Subtasks by due date assigned')
```

Comment:
```suggestion
			.setName('Only show subtasks that are due today')
```

i think something along these lines is more clear